### DISCUSSION TITLE
Nautilus - does not work after the update

### DISCUSSION THREAD

**Discussion by Vyacheslav-S** [7Ji/orangepi5-archlinuxarm#39] [Q&A]
Yesterday, several packages were updated and the Nautilus file manager became non-working. 

Screenshot:

![Снимок экрана от 2024-05-17 10-11-59](https://github.com/7Ji/orangepi5-archlinuxarm/assets/47851954/de2ae4b9-a1e4-4e61-9645-bbffe95fedc9)

Update packages:
```
mesa-panfork-git-r164486.2e8aead0016-1-aarch64.pkg.tar.zst

libv4l-rkmpp-git-1.7.0.r5.fb10c83-1-aarch64.pkg.tar.zst

gnome-session-46.0-2-aarch64.pkg.tar.xz

xdg-desktop-portal-gnome-46.1-2-aarch64.pkg.tar.xz

gdk-pixbuf2-2.42.12-1-aarch64.pkg.tar.xz

xorg-xwayland-23.2.7-1-aarch64.pkg.tar.xz
```

How can this problem be solved?

**Comment by squassina** [comment]
did you try calling it from a terminal and checking where it's failing? (I'm not using it, so I cannot help much without the " logs").

**Reply by Vyacheslav-S** [reply]
No problem when starting from the terminal

```
alarm@opi5p:~$ nautilus
** Message: 09:19:46.977: Connecting to org.freedesktop.Tracker3.Miner.Files
Initializing nautilus-image-converter extension
open-any-terminal: terminal is set to "terminator" opening a new window 
```
But for some reason it doesn't work:

![image](https://github.com/7Ji/orangepi5-archlinuxarm/assets/47851954/f6532c9c-bcb9-41dc-8a20-a039bd8bda0b)

**Comment by Vyacheslav-S** [comment]
Many other applications also have no interface. 
Maybe there is some kind of solution? 
I don't see any problems in the logs.
